test/config.js

```diff
@@ -174,6 +182,12 @@ exports.arrange_args = {
     },
 };
 
+const hjsonRes = {
+    matt: 'waz here',
+    array: [ 'has an element' ],
+    objecty: { 'has a property': 'with a value' }
```

Comment:
The contents of this test file should be different than the contents of the json file, so that we're assuring that the correct file got read and parsed.

Reply:
Ok, understood